modules/drivers/mongo/src/metabase/driver/mongo/util.clj

```diff
@@ -89,6 +88,64 @@
     :else                         (throw (Exception. (str "with-mongo-connection failed: bad connection details:"
                                                           (:details database))))))
 
+(defn- connect-via-uri
+  [^MongoClientURI uri ^MongoClient conn]
+  (if-let [dbName (.getDatabase uri)]
+    (.getDB conn dbName)
+    (throw (IllegalArgumentException. "No database name specified in URI. Monger requires a database to be explicitly configured."))))
+
+(defn- connect
+  "Connects to Mongo.  This is the fallback method to connect to hostnames that are
+   not FQDNs.  This works with 'localhost', but has been problematic with FQDNs.
+   If you would like to provide a FQDN, use `connect-srv`."
```

Comment:
You don't have to change this now, it's not important, but we've been aiming for a line length of around 120 characters because that seems to be what GitHub displays on-screen without cutting things off so it makes looking at PRs nicer. I don't know how you would change it in IntelliJ but if it's a configuration thing you can add to the project dir feel free to do so, we have the equivalent settings for Emacs here https://github.com/metabase/metabase/blob/master/.dir-locals.el#L28

Reply:
Oh yeah, sorry about being nitpicky here, but for next time we usually try to indent docstrings 2 spaces as suggested by the Clojure Style Guide
https://guide.clojure.style/#docstring-indentation.

Not something I would consider to be a reason to hold off on merging, just keep it in mind for next time